root-common/usr/share/container-scripts/mysql/init/50-passwd-change.sh

```diff
@@ -4,24 +4,27 @@ password_change() {
   # Set the password for MySQL user and root everytime this container is started.
   # This allows to change the password by editing the deployment configuration.
   if [[ -v MYSQL_USER && -v MYSQL_PASSWORD ]]; then
-mysql $mysql_flags <<EOSQL
-      ALTER USER '${MYSQL_USER}'@'%' IDENTIFIED BY '${MYSQL_PASSWORD}';
-EOSQL
+mysql $mysql_flags \
+<<EOF
+ALTER USER '${MYSQL_USER}'@'%' IDENTIFIED BY '${MYSQL_PASSWORD}';
```

Comment:
Is there some guard against sql injection?

Reply:
I have updated PR. Of course, I can return back `EOSQL` but I need to store the output of SELECT.

Reply:
EOSQL is back again.